cmd/livepeer/livepeer.go

```diff
@@ -134,8 +134,7 @@ func main() {
 	ethOrchAddr := flag.String("ethOrchAddr", "", "ETH address of an on-chain registered orchestrator")
 	ethUrl := flag.String("ethUrl", "", "Ethereum node JSON-RPC URL")
 	ethController := flag.String("ethController", "", "Protocol smart contract address")
-	gasLimit := flag.Int("gasLimit", 0, "Gas limit for ETH transactions")
-	gasPrice := flag.Int("gasPrice", 0, "Gas price for ETH transactions")
```

Comment:
I'm on board with removing the gasLimit flag, but what do you think about keeping the existing gasPrice flag?

Reasons:

- It is already here and I don't think it is harmful to have
- Leaves the possibility for anyone that already is or that wants to use a fixed default gas price
- Leaves another route to control gas price at least until better options are added (i.e. integrating with a better gas price estimation API)

Reply:
I'm wondering if it makes sense to keep it. 

Let's say you want to use a fixed gas price, are there any benefits that could be gained over using a max gas price instead ? 

Assuming with a fixed gas price you don't want to use a gas price higher than what is set, you can achieve the same thing by setting a max gas price instead. 

Then when the gas price is lower than the max gas price you would still have the benefit of using the lower gas price with similar confirmation times.

Reply:
Sure I personally think the maxGasPrice flag is almost always more useful then a gasPrice flag that sets a fixed gas price. I mainly want to go through the exercise of checking if it makes sense to immediately remove a user facing flag if we don't know if anyone uses them. If it's not harmful to leave these flags for now (now also including -gasLimit in there as well), I think we should consider keeping them and their associated functionality for another release, logging a deprecation warning similar to what is done for the old [S3 object store flags](https://github.com/livepeer/go-livepeer/blob/fd07a5673175aa6d10f56dde6eb3ef2a9f5691b9/cmd/livepeer/livepeer.go#L740) and then removing them in another release.